frontend/src/metabase/admin/performance/components/StrategyForm.tsx

```diff
@@ -341,7 +342,7 @@ const ScheduleStrategyFormFields = () => {
       schedule={schedule}
       scheduleOptions={["hourly", "daily", "weekly", "monthly"]}
       onScheduleChange={onScheduleChange}
-      verb={t`Invalidate`}
+      verb={c("A verb in the imperative mood").t`Invalidate`}
```

Comment:
Some translators have rendered this as an infinitive, but it's an imperative. I'm hoping the context will help